src/main/java/uk/gov/justice/digital/delius/controller/secure/ReferenceDataResource.java

```diff
@@ -89,12 +87,47 @@
             })
     @GetMapping(value = "/probationAreas/code/{code}/localDeliveryUnits/code/{lduCode}/teams")
     public Page<KeyValue> getTeamsForLdu(
-            @ApiParam(name = "code", value = "Probation area code", example = "NO2", required = true)
-            final @PathVariable String code,
-            @ApiParam(name = "lduCode", value = "Local delivery unit code", example = "NO2NPSA", required = true)
-            final @PathVariable String lduCode) {
+            @ApiParam(name = "code", value = "Probation area code", example = "NO2", required = true) final @PathVariable String code,
+            @ApiParam(name = "lduCode", value = "Local delivery unit code", example = "NO2NPSA", required = true) final @PathVariable String lduCode) {
 
         log.info("Call to getTeamsForLdu");
         return referenceDataService.getTeamsForLocalDeliveryUnit(code, lduCode);
     }
+
+    @ApiOperation(value = "Return the reference data items for the given set")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+                    @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
+                    @ApiResponse(code = 404, message = "Data set not found", response = ErrorResponse.class),
+                    @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+            })
+    @GetMapping(value = "/referenceData/set/{set}")
```

Comment:
should really be `reference-data` but doesn't fit with rest of community api

Reply:
yeah agreed, thought it best to keep with tradition